### DISCUSSION TITLE
DO NOT READ LOGS AUTOMATICALLY

### DISCUSSION THREAD

**Discussion by hshen13** [cline/cline#752] [Feature Requests]
Please add a feature to prevent Cline from automatically reading log outputs, as they are often too lengthy and result in unnecessary token usage. Additionally, I suggest providing an option for users to toggle automatic log reading on or off based on their preference.

**Comment by decentralize-all** [comment]
Can add to custom instructions to tail -50 all commands which produce logs. I do agree there should be some max output token detection by CLINE to protect from DDOS'ing the tokenizer of a given LLM.
